packages/grafana-data/src/transformations/fieldReducer.ts

```diff
@@ -271,33 +271,41 @@ export const fieldReducers = new Registry<FieldReducerInfo>(() => [
   },
 ]);
 
+// Used for test cases
+export const defaultCalcs: FieldCalcs = {
+  sum: 0,
+  max: -Number.MAX_VALUE,
```

Comment:
Not totally sure of the context here, but I've often seen Number.MAX_SAFE_INTEGER used instead of Number.MAX_VALUE if mainly integers are mainly at play.

Reply:
I think i might just leave this "as is" given these are the default calcs for the last several years and I don't want to increase surface area of current changes 😬